doc/whats-new.rst

```diff
@@ -12,6 +12,8 @@ v2025.08.1 (unreleased)
 
 New Features
 ~~~~~~~~~~~~
+- Run ``TestNetCDF4Data`` as ``TestNetCDF4DataTree`` through ``open_datatree`` (:pull:`10632`).
+  By `Kai Mühlbauer <https://github.com/kmuehlbauer>`_.
```

Comment:
I would probably omit this. This is an internal improvement to Xarray's own test suite that does not concern users.

Reply:
Yes, most likely not of concern to users and for sure no new feature. I'll move it to the internal section, then.

Reply:
Done in #ed67618.